### ISSUE TITLE
[GitHub] "Could not parse response" import private organization project

### ISSUE THREAD

**Issue by creichert** [smallhadroncollider/taskell#20]
```
$ taskell -g repos/private/private github.md
Fetching project list from GitHub...

[1] test runner
[2] dashboard
[3] ops
[4] api

Import project: 1
Fetching project from GitHub...
Could not parse response. Please file an Issue on GitHub:
https://github.com/smallhadroncollider/taskell/issues/new
```

- What are the limitations in the current GitHub integration? Our project board isn't complicated but uses a lot of references and markdown

**Comment by smallhadroncollider** [comment]
It's the last thing I added before going on parental leave, so there are definitely going to be some issue with it - but I've not had the chance to fully explore what those might be.

I'm on parental leave for the next few months, but I'll look into this once I'm back.

**Comment by creichert** [comment]
> I'm on parental leave for the next few months, but I'll look into this once I'm back.

YES, please don't worry about me, I'm happy to hear you're taking that time. I'll hopefully have some time to check this out myself ASAP.

**Comment by smallhadroncollider** [comment]
As a first step I'm going to try and improve the error reporting so that it will at least tell you specifically what's gone wrong. Once that's in it should be possible to fix whatever is causing the error in the first place.

**Comment by smallhadroncollider** [comment]
Right, in [1.3.4](
https://github.com/smallhadroncollider/taskell/releases/tag/1.3.4) I've added better error reporting to the API requests. Hopefully that will tell us why it can't parse the response.

If you could get 1.3.4 at some point and then let me know the error you get back I'll be able to look into this properly. Thanks.

**Comment by smallhadroncollider** [comment]
Think this is sorted now [#62]
